spec/travis/yml/accept/virt_spec.rb

```diff
@@ -0,0 +1,69 @@
+describe Travis::Yml, 'virt' do
+  subject { described_class.apply(parse(yaml), opts) }
+
+  describe 'no default' do
+    yaml ''
+    it { should serialize_to empty }
+  end
+
+  known = %w(
+    lxd
+    vm
+  )
+
+  known.each do |value|
+    describe "given #{value}" do
```

Comment:
similar rule should be for:
arch: amd64
virt: #{value}

Reply:
removed that for now, @michal-at-travisci